[BSv5] Refactor Click-to-copy (#1447)
google/docsy: userguide/content/en/docs/adding-content/lookandfeel.md

{{< /tab >}}
{{< /tabpane >}}

> If you are using a Docsy version later than `0.6.0`, the code blocks show a

Reviewer: ```suggestion
If you are using a Docsy version `0.6.0` or later, code blocks show a
```

Author: This does not work for `0.6.0` and we haven't definitely decided on the next version number. I think "later than" covers that. About the article "the", although I know  I don't really need it here, I try to never omit it, based on this guideline: https://developers.google.com/style/articles

"Copy to clipboard" icon in the top right-hand corner. To disable this

> functionality add the `disable_click2copy_chroma` option to your
> configuration file, under `params`:
> 
> {{< tabpane persistLang=false >}}
> {{< tab header="Configuration file:" disabled=true />}}
> {{< tab header="hugo.toml" lang="toml" >}}
> disable_click2copy_chroma = true
> {{< /tab >}}
> {{< tab header="hugo.yaml" lang="yaml" >}}
> disable_click2copy_chroma: true
> {{< /tab >}}
> {{< tab header="hugo.json" lang="json" >}}
> "disable_click2copy_chroma": true
> {{< /tab >}}
> {{< /tabpane >}}

Reviewer: I'm not a fan of the use of these manually coded TOML/YAML/JSON blocks, esp. when the configuration parameter is so simple:

```suggestion
functionality set `params.disable_click2copy_chroma` to `true` in your
configuration file.
```

Author: Completely agree. I was mostly following what's already in this doc. Updated.
